### PR TITLE
feat: ILS usage graphs

### PR DESCRIPTION
**Pull request by Chloe070196**
Description

An Aspen admin can now view a usage graph and a raw data table for each of the variables included in the ILS Integration dashboard. This feature used to be limited to the System Reports dashboard, and it is now being extended.

Note: the implementation is based on the the System Reports existing usage graphs, and intends to mirror it where possible

Features:

The dashboard now includes links next to the title of each of its sections
Those links lead to the relevant Usage Graph page
The Usage Graph page contains both a graph and a raw data table
While the user is on a given ILS Integration Usage Graph page, the ILS Integration section on the left hand-side options menu remains open (active)

Test plan:

- apply the patch
- log in as an admin
- navigate to Administration Settings > ILS Integration > Dashboard
- click on the link displayed next to a section's header (title) and access the relevant Usage Graph page
- notice that the data displayed matches the summary found on the dashboard

### CODE REVIEW

File: code/web/interface/themes/responsive/ILS/dashboard.tpl [hunk 1]
```diff
@@ -31,11 +30,11 @@
 						</div>
 					</div>
 				</div>
-				
+
 				<div class="dashboardCategory col-sm-6">
 					<div class="row">
 						<div class="col-sm-10 col-sm-offset-1">
-							<h2 class="dashboardCategoryLabel">{translate text="Self Registrations" isAdminFacing=true}</h2>
+							<h2 class="dashboardCategoryLabel">{translate text="Self Registrations" isAdminFacing=true} <a href="/ILS/UsageGraphs?stat=selfRegistrations&instance={$selectedInstance}" title="{translate text="Show User Logins Graph" inAttribute="true"}"><i class="fas fa-chart-line"></i></a></h2>
```
Comment (mdnoble73) [inline code/web/interface/themes/responsive/ILS/dashboard.tpl]: These all have the same title text.  Can you update to use distinct title text for each link?  Also they should have isAdminFacing set to true in addition to inAttribute

**Review by mdnoble73 — CHANGES_REQUESTED** [review]
Check the title text for the new dashboard links

File: code/web/interface/themes/responsive/ILS/dashboard.tpl (hunk 1 above)
Comment (Chloe070196) [reply code/web/interface/themes/responsive/ILS/dashboard.tpl]: Apologies for the oversight, will do!

Comment (mdnoble73) [reply code/web/interface/themes/responsive/ILS/dashboard.tpl]: No worries, thanks for the fix!

**Review by mdnoble73 — CHANGES_REQUESTED** [review]
I'm, getting SQL errors with the following graphs.  

- recordsHeld
- pdfsDownloaded
- pdfsViewed
- supplementalFilesDownloaded
- totalHolds
They are all in the form of: 

> SQLSTATE[42000]: Syntax error or access violation: 1140 In aggregated query without GROUP BY, expression #1 of SELECT list contains nonaggregated column 'model.ils_record_usage.year'; this is incompatible with sql_mode=only_full_group_by

**Review by mdnoble73 — APPROVED** [review]

**Pull request merged**